docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,565 @@
-_imageVisual = VisualFactory.Get().CreateVisual(imageVisual);
+The N-patch image visual renders an N-patch or a 9-patch image. It uses non quad geometry. Both geometry and texture are cached to reduce memory consumption, if the same N-patch image is used elsewhere.
 
-RegisterVisual(ImageVisualPropertyIndex, _imageVisual);
+Properties specific for the NPatchVisual are listed in the table below:
 
-/// Set the image visual depth index
-_imageVisual.DepthIndex = ImageVisualPropertyIndex;
-```
+**Table: NPatchVisual properties**
 
-<a name="npatch"></a>
-### N-Patch Image
+| Property            | Type               | Required | Description                                                     |
+|---------------------|--------------------|----------|-----------------------------------------------------------------|
+| `URL`               | `string`           | Yes      | The URL of the image.                                           |
+| `BorderOnly`        | `bool`             | No       | If `true` only borders are drawn (default is `false`).          | 
+| `Border`            | `Rectangle`        | No       | The border of the image in the order - left, right, bottom, top |
 
-The n-patch image visual renders an n-patch or a 9-patch image. It uses non quad geometry. Both geometry and texture are cached to reduce memory consumption, if the same n-patch image is used elsewhere.
 
-**Figure: N-patch image visual**
 
-![N-patch image visual](media/n-patch-visual.png)
+**Usage:** 
 
-The visual map for an n-patch image is `NPatchVisual`.
+The code illustrates how to plot N-Patch in NUI (the absolute path to the image is set as in case of the [`ImageVisual`](#imageVisual)):
+   ```csharp
+   string _imageUrl = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
 
-<a name="svg"></a>
-### SVG Image
+   NPatchVisual _nPatchVisual = null;
 
-The SVG image visual renders an SVG image into the control's quad. It supports the following features from the [SVG Tiny 1.2 Specification](https://www.w3.org/TR/SVGTiny12):
+   _nPatchVisual = new NPatchVisual();
+   _nPatchVisual.URL = _imageUrl + "heartsframe.png";
+   _nPatchVisual.RelativeSize = new RelativeVector2(1.0f, 0.3f);
+   _nPatchVisual.Origin = Visual.AlignType.BottomEnd;
+   _nPatchVisual.AnchorPoint = Visual.AlignType.BottomEnd;
+
+   _visualView.AddVisual("NPatch-Image1", _nPatchVisual);
+
+   _nPatchVisual = new NPatchVisual();
+   _nPatchVisual.URL = _imageUrl + "heartsframe.png";
+   _nPatchVisual.RelativeSize = new RelativeVector2(0.3f, 0.6f);
+   _nPatchVisual.Origin = Visual.AlignType.TopCenter;
+   _nPatchVisual.AnchorPoint = Visual.AlignType.TopCenter;
+
+   _visualView.AddVisual("NPatch-Image2", _nPatchVisual);
+
+   ```
+
+**Figure: NPatchVisual from the code above - the correct scaling of the NPatch image is shown (_visualView background color set to gray).**
+
+![NPatchVisual](media/visuals/NPatchVisualExample.png)
```

Comment:
Thank you for adding images to help understand. But how about using a more calm image?

Reply:
I'll try to create or find one

Reply:
since I closed this pr and opened a new one, I would like to comment that I remember your request to change the picture. I will do it in a new pr in which I will extend the picure related part.